CODEOWNERS

```
@@ -171,6 +171,7 @@
 /bundles/org.openhab.binding.mqtt.homie/ @davidgraeff
 /bundles/org.openhab.binding.mystrom/ @pail23
 /bundles/org.openhab.binding.nanoleaf/ @raepple
+/bundles/org.openhab.binding.nanoleaf/ @stefan-hoehn
```

multiple codeowners should be added space separated:
```suggestion
/bundles/org.openhab.binding.nanoleaf/ @raepple @stefan-hoehn
```